src/Components/Footer.jsx

```diff
@@ -1,18 +1,59 @@
+import React from "react";
+import { Grid, Box, Container } from "@material-ui/core";
```

Comment:
let's combine the imports that are coming from the same libraries. i.e. `import { Grid, Box, Container, List, ListItem } from "@material-ui/core";` same thing with the imports from `@mui/material`